.gitignore

```diff
@@ -2,10 +2,74 @@
 .envrc
 
 # IDEs
-.idea/
 .vscode/
 
 # Compiled binary
 /alloydb-auth-proxy
 
 /key.json
+
+# Google App Engine generated folder
+appengine-generated/
+
+# Java
+*.class
+
+# Mobile Tools for Java (J2ME)
+.mtj.tmp/
+
+# Package Files #
+*.jar
+*.war
+*.ear
+
+# virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
+hs_err_pid*
+
+# jenv
+.java-version
+
+# maven
+target/
+pom.xml.tag
+pom.xml.releaseBackup
+pom.xml.versionsBackup
+pom.xml.next
+release.properties
+dependency-reduced-pom.xml
+buildNumber.properties
+.checkstyle
+
+# gradle
+build/
+
+# Secrets
+service-account.json
+secrets.env
+
+# intellij
+.idea/
+*.iml
+
+# Eclipse files
+.project
+.classpath
+.settings
+
+# vim
+[._]*.s[a-w][a-z]
+[._]s[a-w][a-z]
+Session.vim
+.netrwhist
+*~
+tags
+
+# Gradle
+out/
+.gradle/
+
+# VS Code
+.vscode/
+
+# OSX
+.DS_Store
```

Comment:
Can we combine all of these in a section for "samples" since we will want to remove them eventually?

Reply:
fixed.